content/administration/maintain/online.rst

```diff
@@ -37,12 +37,29 @@ Upgrade
 
 Upgrade the database to the latest Odoo version.
 
+In your database you will be invited to upgrade if you are not on the latest version we support on
+the SaaS.
+
+- If you are currently on a long-term supported version of Odoo
+
+  - The upgrade button will propose an upgrade to the latest version (e.g: 13 to 15).
+  - If you are already on the latest version, you will be invited to upgrade to the last Online
+    Version but you can dismiss the request and stay on the latest supported version. You won’t
```

Comment:
Is "Online Version" what's defined as "SaaS Version" in the supported versions page?
I'll change it to "SaaS version" to stay consistent, but let me know if I misunderstood, @apr-odoo